### PR TITLE
Authenticaton

### PR DESCRIPTION
**Pull request by KaustubhM1997**
Added Auth and corresponding toasts

### CODE REVIEW

File: src/pages/Login/login.css [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  border: 1px solid;
+  width: 25rem;
+  margin-left: auto;
+  margin-right: auto;
+  border: none;
+  box-shadow: var(--full-box-shadow);
+  padding-left: 2rem;
+  padding-right: 2rem;
+  border-radius: 5px;
+  position: relative;
```
Comment (himadri2110) [inline src/pages/Login/login.css:23]: Suggestion: You can use shorthand properties for `margin` and `padding`.

File: src/pages/Login/login.js [hunk 2]
```diff
@@ -1,6 +1,115 @@
...
+      });
+
+      setErrors(""); // no errors initially
+
+      //sending data to backend using post that we get from user and receiving a response
+
+      const response = await axios.post("/api/auth/login", {
+        email: input.email, ///we send these values to the backend that we get from the user to check whether this user exists. The test credentials are adarsh balika, this user exists on the backend.
+        password: input.password,
+      });
```
Comment (himadri2110) [inline src/pages/Login/login.js:41]: You can make a separate services file for all the API calls

File: src/pages/Signup/signup.js [hunk 3]
```diff
@@ -1,6 +1,142 @@
...
+    }
+  };
+
+  return (
+    <div class="signup-container">
+      <h2 class="login-header">Signup</h2>
+
+      <div class="login-details">
+        <div class="login-subdetails">
+          <p class="subdetails-header">Name</p>
```
Comment (himadri2110) [inline src/pages/Signup/signup.js:65]: Please replace `class` with `className`

File: src/services/signoutService.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  localStorage.removeItem("TOKEN");
+  setAuth({
+    Authenticated: false,
+    token: "",
+  });
+  toast.error("You have logged out");
+  navigate("/");
+};
+
+export { signOut };
```
Comment (himadri2110) [inline src/services/signoutService.jsx:12]: This isn't really a service, I guess. You are just removing the item from localStorage and resetting the state. Only API calls should go inside a service file.

**Review by himadri2110 — APPROVED** [review]
Looks good for now. 👍🏻 
You can refactor and optimize later.

**Review by someshtiwarii — APPROVED** [review]
other than what @himadri2110  suggested, rest looks fine.

**Pull request merged**